frontend/src/scenes/persons/RelatedFeatureFlags.tsx

```diff
@@ -83,6 +83,8 @@ export function RelatedFeatureFlags({ distinctId, groups }: Props): JSX.Element
                     <div style={{ wordBreak: 'break-word' }}>
                         {featureFlag.active && featureFlag.value
                             ? capitalizeFirstLetter(featureFlag.value.toString())
+                            : !featureFlag.active
```

Comment:
this returns `--` for out of rollout bound error, and for no group type.

I think we can just return `False` instead of '--', not sure where '--' makes sense anymore.

Reply:
yeah i'd agree here!